packages/dialtone-vue3/components/scroller/scroller.vue

```diff
@@ -33,6 +33,10 @@ import CoreScroller from './modules/core_scroller.vue';
 import DynamicScroller from './modules/dynamic_scroller.vue';
 import { provide, computed, watch, ref } from 'vue';
 
+defineOptions({
+  name: 'DtScroller',
+});
```

Comment:
Oh nice, don't think there was a way to do this when we first wrote this.

Reply:
I think previously you could achieve this by creating another `<script>` tag which is very awful IMO haha, this way is more clear